src/box/txn.c

```diff
@@ -45,6 +45,15 @@ double too_long_threshold;
 /** Last prepare-sequence-number that was assigned to prepared TX. */
 int64_t txn_last_psn = 0;
 
+enum txn_isolation_level default_isolation_level = TXN_ISOLATION_BEST_EFFORT;
+
+const char *txn_isolation_level_strs[txn_isolation_level_MAX] = {
+	"default",
+	"read-committed",
+	"read-confirmed",
+	"best-effort",
+};
```

Comment:
AFAIC, these are only used in Lua via FFI: perhaps this array is redundant and the enum -> string conversion should be done in Lua?